docs/usage/python.md

```diff
@@ -14,22 +14,25 @@ Renovate supports the following Python package managers:
 
 ## Versioning support
 
-The [PEP440](https://www.python.org/dev/peps/pep-0440/) versioning scheme has been rewritten for JavaScript for the purposes of use in this project is published as [@renovatebot/pep440](https://github.com/renovatebot/pep440).
-It supports both pinned versions as well as ranges.
-Legacy versions (`===` prefix) are ignored.
+We've written a JavaScript version of the [PEP440 specification](https://www.python.org/dev/peps/pep-0440/) so we can use it in Renovate bot.
+You can find this project in our [`renovatebot/pep440` repository](https://github.com/renovatebot/pep440).
+
+Our PEP440 implementation supports pinned versions and ranges.
+Legacy versions with the `===` prefix are ignored.
 
 ## How it works
 
 1. Renovate searches through each repository for package files
 1. Existing dependencies are extracted from the package files
-1. Renovate looks up the latest version on [PyPI](https://pypi.org/) to determine if any upgrades are available
-1. If the source package includes a GitHub URL as its source, and has a "changelog" file or uses GitHub releases, a Release Note will be embedded in the generated PR
+1. Renovate searches for the latest version on [PyPI](https://pypi.org/) to decide if there are upgrades
+1. If the source package includes a GitHub URL as its source, and has a "changelog" file _or_ uses GitHub releases, a Release Note will be embedded in the generated PR
 
 ## Alternative file names
 
-The default file matching regex for `requirements.txt` aims to pick up the most popular conventions for file naming, but it's possible that some get missed.
-If you have a specific file or file pattern you want the Renovate bot to find, use the `fileMatch` field in the `pip_requirements` object.
-e.g.:
+The default file matching regex for `requirements.txt` follows common file name conventions.
```

Comment:
this is only true for our `pip_requirements` manager. we should probably move this section to the manager readme